File: python/logic2/no_teen_sum.py
def fix_teen(n):
    if (n == 15):
        return 15
    elif (n == 16):
        return 16
    elif (n >= 13 and n <= 19):
        return 0
    else:
        return n


def no_teen_sum(a, b, c):
    return fix_teen(a) + fix_teen(b) + fix_teen(c)

File: java_code/logic2/noTeenSum.java
package java_code.logic2;

public class noTeenSum {
  public int noTeenSum(int a, int b, int c) {
    int sum = 0;

    if (a >= 13 && a <= 19)
      sum += fixTeen(a);
    else
      sum += a;

    if (b >= 13 && b <= 19)
      sum += fixTeen(b);
    else
      sum += b;

    if (c >= 13 && c <= 19)
      sum += fixTeen(c);
    else
      sum += c;

    return sum;

  }

  public int fixTeen(int n) {
    if (n == 15) {
      return 15;
    } else if (n == 16) {
      return 16;
    } else
      return 0;
  }
}